js-task/js/index.js

```diff
@@ -0,0 +1,16 @@
+function newString() {
+    const str = prompt("Please enter a string of 3 or more characters:");
+
+    // handling the string characters (three or more)
+
+    if (str.length < 3) {
+      return "String length must be 3 or more.";
+    }
+    const lastThree = str.slice(-3);
+
```

Comment:
Instead of using lastThree, a more descriptive name like lastThreeChars would be a better alternative